### DISCUSSION TITLE
1panel安装redis失败

### DISCUSSION THREAD

**Discussion by zp9611** [1Panel-dev/1Panel#6657] [Q&A]
版本：1panel 专业版v1.10.18-lts，redis 7.4.0
现象：点击安装redis后自动放回主页概览，重新打开应用商城没有安装成功和过程，在日志审计=》操作日志、系统日志内均没有关于安装redis的日志
主机状态：4核（7.88%），7.54GB（29.32%），1.33%负载，177.05GB（9.8%），所以不存在硬件不支持

社区发帖问过了，快两周了，没有回复，希望有人帮忙确认是哪的问题，我现在是不知道从哪解决

**Comment by wanghe-fit2cloud** [comment]
1. 使用 `IP:端口` 方式访问 1Panel 时是否会遇到上述问题？
2. 如果是通过 `反向代理` 网站方式访问 1Panel，可能是被 WAF 拦截了（可以通过查看拦截记录确认）

**Reply by zp9611** [reply]
非常感谢给出的解决方向，经过排查是前面的一层waf判断异常威胁命令拦截了，使用IP登录面板可以正常安装redis
